### ISSUE TITLE
Fingerprint Assets?

### ISSUE THREAD

**Issue by gblakeman** [CunningFolkDev/elevate360_website#49]
Use something like [jekyll-assets](https://github.com/envygeeks/jekyll-assets) to properly fingerprint asset filenames at deployment.

**Comment by gblakeman** [comment]
@lukemiller I ran into a nasty caching issue today when I made some css changes. The culprit is that the css filename doesn't change on deploy and the CDN is telling browsers to cache it forever, basically.

Would you be open to altering the assets setup in Jekyll to have it automagically fingerprint the asset names with each change?

I’d be happy to do the setup. I’m familiar with `jekyll-assets` and once it’s configured it shouldn’t affect your work at all.

The alternative is that we remember to [manually change filenames](https://github.com/CunningFolkDev/elevate360_website/pull/48) whenever we make a big change to an image, js, or css file. It works, but it’d be a pain.

Thoughts?

**Comment by lukemiller** [comment]
yeah, we were manually invalidating the cache before to get around that, but that's also a pain.

I haven't used jekyll-assets before, but if you're happy with it, that's fine with me.

There is also this post I found, but it seems a bit tedious to set up:  http://brettterpstra.com/2014/02/21/a-jekyll-cdn-with-cloudfront/

**Comment by gblakeman** [comment]
Cool, I’ll make that transition when I have some time. It won’t take me long and then the whole thing will be resolved and automatic.
